docs/meetings/events.md

```diff
@@ -0,0 +1,123 @@
+(events)=
+# Events and meetings
+
+(collab-cafe)=
+## Collaboration Cafés
+
+On the first and third Tuesdays of every month, the [JupyterHub community](https://compass.hub.jupyter.org/meetings/collab-cafe/) and JupyterBook communities host a shared Collaboration Café.
+
+These community events are designed to enable community members to work together and facilitate discussion on any topics they need in breakout rooms.
+They are open for anyone to attend.
+
+You can read more about why and how we host Collaboration Cafés in the resources below.
+
+- [JupyterHub blog post](https://blog.jupyter.org/online-collaboration-caf%C3%A9-launch-jupyterhub-team-meetings-to-become-more-collaborative-spaces-b713edadf15)
+- [JupyterBook blog post](xref:blog/posts/2025-04-09-new-community-meeting) 
+- [_The Turing Way_ Collaboration Café guide](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html)
+
+## Quick info
+
+- Link to running notes: [https://mypads2.framapad.org/p/jupyterhub-jupyterbook-collaborationcafe-nr53am9wz](https://mypads2.framapad.org/p/jupyterhub-jupyterbook-collaborationcafe-nr53am9wz)
```

Comment:
Any reason for the item lists to be "Link to ..." followed by the link instead of the more short version used in the previous list?